modules/core/client/controllers/groupController.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+angular.module('core').controller('GroupCtrl', function ($scope, $log) {
```

Comment:
!fix Can you please comment on what 'log' is? Just comment the code a little better about what is happening here